### ISSUE TITLE
[Feature]: Add appearance preview

### ISSUE THREAD

**Issue by DylanBT928** [movie-web/movie-web#868]
### What feature do you want to add?

Change the background colors when you change the appearance settings in order to know what it looks like without pressing "save" everytime.

### Why do you want to have this feature?

This feature would be a lot more convenient to users when wanting to change their background colors. Having to choose a new appearance then clicking "save" to view what it looks like may be tedious if they do not like that color and decide to change it again.

**Comment by DylanBT928** [comment]
Do I need Docker to preview/run the website before committing? Docker won't work for me.

**Comment by zisra** [comment]
No, you don't. You would need to:
1. Install dependencies (`pnpm install`)
2. Deploy locally (`pnpm run dev`), then open the given link

**Comment by marcoslor** [comment]
Hi there!
I think this project is reeealy awesome and I'd really like to start contributing to it, and I found this feature request could be a nice start, so I've created #922 to implement this feature.

Some things I took into consideration while implementing it:
- It would be nice for the user to know which theme is in use, and which is just being previewed.
- A nice little CSS transition would be cool (I think only to the background is enough).
- The preview should only be seen while the user is in the /settings page. 

I've included a small demo video on my pull request.
